src/tests/components/Tip.test.tsx

```diff
@@ -17,7 +17,7 @@ describe('Tip component', () => {
     jest.clearAllMocks();
   });
 
-  const calculateTip = (perecentage) => (perecentage * total).toFixed(2);
+  const calculateTip = (perecentage) => parseFloat((perecentage * total).toFixed(2));
```

Comment:
```suggestion
  const calculateTip = (percentage) => parseFloat((perecentage * total).toFixed(2));
```
also, should this be type annotated?

Reply:
If I must...